config/config.go

```diff
@@ -260,16 +358,10 @@ func includeConfigFile(config *Config, include string) error {
 			config.Roles = rolesSaved
 		}
 
-		for kind, plugins := range config.Plugin {
-			for key, conf := range plugins {
-				if pluginSaved[kind] == nil {
-					pluginSaved[kind] = PluginConfigs{}
-				}
-				pluginSaved[kind][key] = conf
-			}
+		// Overwrite plugin configurations.
```

Comment:
Yes, I understand that. but old code explicitly saves plugins and restores them.
So, I'm confused at first. maybe just adding "same plugin name of configurations" is fine.